tasks/zkml-convert-model/task.yaml

```diff
@@ -0,0 +1,36 @@
+id: zkml-model-conversion
+description: Converts a onnx model to an EZKL circuit. Also copies all input model files to the output model allowing for endpoint creation
+is_active: true
+project_type: 8
+param_groups:
+- name: inputs
+  params:
+  - name: onnxModel
+    description: ONNX model
+    value: null
+    data_type: model
+    required: true
+- name: outputs
+  params:
+  - name: outputCircuit
+    description: The dataset to which this TaskRun will output it's results
+    value: null
+    data_type: model
+    required: false
+- name: parameters
+  params:
+  - name: inputVisibility
```

Comment:
I'd rename this to `privateInput` or `publicInput`. Can you do this for other 2 visibility params as well?